components/layout_2020/flow/inline.rs

```diff
@@ -393,6 +391,118 @@ impl<'box_tree> PartialInlineBoxFragment<'box_tree> {
     }
 }
 
+fn layout_atomic<'box_tree>(
+    layout_context: &LayoutContext,
+    ifc: &mut InlineFormattingContextState<'box_tree, '_>,
+    atomic: &'box_tree IndependentFormattingContext,
+) {
+    let cbis = ifc.containing_block.inline_size;
+    let padding = atomic.style.padding().percentages_relative_to(cbis);
+    let border = atomic.style.border_width();
+    let margin = atomic
+        .style
+        .margin()
+        .percentages_relative_to(cbis)
+        .auto_is(Length::zero);
+    ifc.inline_position += padding.inline_start + border.inline_start + margin.inline_start;
```

Comment:
AFAICT, we always add together the three same values from `padding`, `border` and `margin`, consider introducing a local `pbm` variable as we do in the other layout functions.

Reply:
Done.